<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentDash</class>
 <widget class="QMainWindow" name="StudentDash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>5a0167cfd7045febc5828df2b5cfbb97.png</normaloff>5a0167cfd7045febc5828df2b5cfbb97.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Sbutton4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>510</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>Numbers Test</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Sbutton2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Alphabet Test</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Sbutton3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>Learn Numbers</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Sbutton1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jokerman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 0);</string>
    </property>
    <property name="text">
     <string>Learn Alphabet</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="HomeButton1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>490</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>house-icon.png</normaloff>house-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>521</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(mysaline-asl.jpg);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(cooltext372373214987174.png);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
